internal/featuresupport/feature_support_level.go

```diff
@@ -17,7 +17,6 @@ var featuresList = map[models.FeatureSupportLevelID]SupportLevelFeature{
 	models.FeatureSupportLevelIDSINGLENODEEXPANSION:       (&SingleNodeExpansionFeature{}).New(),
 	models.FeatureSupportLevelIDMINIMALISO:                (&MinimalIso{}).New(),
 	models.FeatureSupportLevelIDFULLISO:                   (&FullIso{}).New(),
-	models.FeatureSupportLevelIDSKIPMCOREBOOT:             &skipMcoReboot{},
```

Comment:
Should we remove it also from swagger or add at least comment?

Reply:
I removed it, did i miss anything?

Reply:
Sorry, missed that :)